.changeset/five-hoops-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-hive/cli': minor
+---
+
+Improve output of the `hive whoami` command. It now also handles the new access token format.
```

Comment:
Also should provide a changeset for `hive` for the personal access tokens.

Reply:
I added precise changesets, more details is covered in https://github.com/graphql-hive/console/pull/7231